packages/fuchsia_ctl/tool/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2019 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+command -v cipd > /dev/null || {
+  echo "Please install CIPD (available from depot_tools) and add to path first.";
+  exit -1;
+}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)"
+
+cipd ensure -ensure-file $DIR/ensure_file -root $DIR
+
+pushd $DIR/..
+
+# rm -rf build
```

Comment:
Intended to keep this comment?

Reply:
Replacing it with a check that the directory doesn't exist.  It was annoying to have it removed, but it should be cleaned before building.